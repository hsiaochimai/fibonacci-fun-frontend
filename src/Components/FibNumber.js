import React from 'react'
export default function FibNumber(props){
return(
    <div className="fib-number">
        <p>{props.num}</p>
    </div>
)
}