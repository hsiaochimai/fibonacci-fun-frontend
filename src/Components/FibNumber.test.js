import React from 'react';
import TestRenderer from 'react-test-renderer';

import FibNumber from './FibNumber';
it('render correctly FibNumber component', () => {  
    const FibNumberComponent = TestRenderer.create(<FibNumber />).toJSON();
    expect(FibNumberComponent).toMatchSnapshot();
});

