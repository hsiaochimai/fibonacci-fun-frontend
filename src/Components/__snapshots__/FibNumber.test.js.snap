// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly FibNumber component 1`] = `
<div
  className="fib-number"
>
  <p />
</div>
`;
